pkg/capr/planner/planner.go

```diff
@@ -62,8 +62,9 @@ const (
 	TLSCertFileArgument                           = "tls-cert-file"
 	DeleteFileAction                              = "delete"
 
-	authnWebhookFileName = "kube-api-authn-webhook.yaml"
-	ConfigYamlFileName   = "/etc/rancher/%s/config.yaml.d/50-rancher.yaml"
+	authnWebhookFileName   = "kube-api-authn-webhook.yaml"
+	ConfigYamlFileName     = "/etc/rancher/%s/config.yaml.d/50-rancher.yaml"
+	JoinServerYamlFileName = "/etc/rancher/%s/config.yaml.d/25-server-bootstrap.yaml"
```

Comment:
Could you add a comment explaining the purpose of this config file, particularly how it helps avoid unnecessary node draining?